roles/setup_minio/README.md

```diff
@@ -55,6 +57,7 @@ See below how to consume the services provided by Minio.
 
 ## From outside the cluster:
 
+> NOTE: The variable `sm_service_type: LoadBalacer` is required to enable external access to the service.
```

Comment:
Sorry I didn't see there was another line below this that I assume it shouldn't be part of the NOTE, in that case it needs a new line:
```suggestion
> NOTE: The variable `sm_service_type: LoadBalacer` is required to enable external access to the service.

```

If the line below is also part of the NOTE then disregard this suggestion

Reply:
np, and correct the lines below should not be part of the NOTE, they are related but it's a separate sentence, so I'm adding the new line.